### DISCUSSION TITLE
Configuration file for defaults

### DISCUSSION THREAD

**Discussion by nicos68** [foxfriends/syncat#16] [Ideas]
Hi,

thanks a lot for your work, the results it produces are clearly superior to what bat does in terms of syntax highlighting. I'd like to add the ability to configure the defaults (mainly numbering, wrapping, framing, but actually any existing option) in the configuration file. Would you accept a PR for that ?

**Comment by foxfriends** [comment]
Yes, that sounds good. Definitely accepting PRs!
